css/src/components/admin-list.css

```diff
@@ -1,45 +1,68 @@
 /**
- * Admin lists.
+ * @file
+ * Admin list and admin item modules.
  */
-ul.admin-list {
-  margin: 0;
+
+/**
+ * Admin list (wrapper of admin items).
+ */
+.admin-list {
+  margin-top: 1em;
+  margin-bottom: 2em;
   padding: 0;
 }
-.admin-list li {
+.admin-list--panel {
+  margin: -1em -1.5em;
+}
+
+/**
+ * Admin item.
+ */
+.admin-item {
+  padding: 0.75em 1.5em 0.75em 3em;
+}
+[dir="rtl"] .admin-item {
+  padding-right: 3em;
+  padding-left: 1.5em;
+}
+.admin-item + .admin-item {
+  border-top: calc(1em / 16) solid var(--color-lightgray);
+}
+.admin-item__title {
+  font-weight: bold;
+}
+.admin-item__title a {
```

Comment:
Do you think we could reasonably add class to the link element?

Reply:
@lauriii Since it's an instance of [GeneratedLink](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21GeneratedLink.php/class/GeneratedLink/8.6.x), i think we have only two options:
* regenerate the link with the deprecated [`Drupal::l()`](https://api.drupal.org/api/drupal/core%21lib%21Drupal.php/function/Drupal%3A%3Al/8.6.x) function
* regenerate the link with `Drupal\Core\Link` (this will change the object's type of course)

Do you have a better idea?

Reply:
@lauriii `admin-list__link` element class added for the link (and the `.admin-item__title a` selectors are replaced).

Reply:
Thank you!